src/lib.rs

```diff
@@ -2189,6 +2189,11 @@ impl FilterLevel {
     pub fn min() -> Self {
         FilterLevel::Off
     }
+
+    /// Check if message with given level should be logged
+    pub fn is_accepted(self, level: Level) -> bool {
```

Comment:
"is accepted" guests (at least IMO) that "self is accepted by level". I would name it ... `does_accept`, `accepts`?

Reply:
That's right, `accepts` is much better for readability. Fixed